elyra/pipeline/component_parser_kfp.py

```diff
@@ -32,6 +32,8 @@ def get_adjusted_component_id(self, component_id):
 
     def parse(self, registry_entry) -> List[Component]:
```

Comment:
Please update the return type hint to -> Optional[List[Component]]: since None can now be returned.